catalog/task/push-sbom-to-pyxis/0.1/README.md

```diff
@@ -6,7 +6,11 @@ Tekton task that extracts sbom from a pull spec and pushes it to Pyxis.
 
 | Name | Description | Optional | Default value |
 |------|-------------|----------|---------------|
-| imageURL | Image URL | No | - |
+| imageURLs | Space separated list of binary image URLs | No | - |
```

Comment:
these binary image URLs contain the SBOM?

Reply:
@scoheb they do, but that's not how we get them. The build pipeline will call `cosign attach sbom` to push a separate image with just the sbom. See here: https://github.com/redhat-appstudio/build-definitions/pull/288/files#diff-3582b1f53880964d98bc28af3a4c933d29c16f79436d21cca84e481e410c675cR271

This is an example from my testing:
```
cosign attach sbom --sbom ~/tmp/sbom/sbom-cyclonedx.json --type cyclonedx quay.io/mmalina/ubi9:latest
WARNING: Attaching SBOMs this way does not sign them. If you want to sign them, use 'cosign attest --predicate /Users/mmalina/tmp/sbom/sbom-cyclonedx.json --key <key path>' or 'cosign sign --key <key path> --attachment sbom <image uri>'.
Uploading SBOM file for [quay.io/mmalina/ubi9:latest] to [quay.io/mmalina/ubi9:sha256-648ae21c84ae3c2e747af5c981674606324b659dcc15c522717ba8d77dcc178e.sbom] with mediaType [application/vnd.cyclonedx+json].
```

You can see the binary image and then the sbom image here: https://quay.io/repository/mmalina/ubi9?tab=tags

So under the hood, cosign will create a new tag for this sbom image. You can customize what tag is used, but both build and us will use the default, so it should just work.

Reply:
maybe I would add this text to description: `Space separated list of binary image URLs that are used to obtain SBOMs via cosign`?

Reply:
Ok, sounds good. Fixed.